src/PowerShell/PowerShellManager.cs

```diff
@@ -23,6 +23,8 @@ internal class PowerShellManager
         private readonly ILogger _logger;
         private readonly PowerShell _pwsh;
 
+        private const string ProfileFileName = "profile.ps1";
```

Comment:
No need to have this const string. It's not used anywhere else. You can directly use it below.

Reply:
Fixed.